codedigger/atcoder/utils.py

```diff
@@ -0,0 +1,55 @@
+import re
+import json
+import requests
+from bs4 import BeautifulSoup as bs4
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+
+from .scrapers import get_user_history
+from .api import get_user_results
+
+from lists.utils import get_next_url, get_prev_url, get_total_page
+from user.exception import ValidationException
+from codeforces.api import user_status
+
+
+def atcoder_status(handle):
+
+    res = get_user_history(handle)
+
+    contests_details = set()
+    all_contest = set()
+    solved = set()
+    wrong = set()
+
+    if res.status_code != 200:
+        return (contests_details, all_contest, solved, wrong)
+
+    soup = bs4(res.content, 'html5lib')
+    contestTable = soup.find('table', {'id': 'history'})
+    del soup
+    if contestTable != None:
+        contests = contestTable.find('tbody').findAll('tr')
+        del contestTable
+        for contest in contests:
+            contests_details.add(
+                contest.findAll('td')[1].find('a')['href'].split('/')[-1])
+        del contests
+
+    res = get_user_results(handle)
```

Comment:
It will raise Validation exception if res.status code is not okay,  handle that.

Reply:
Use TRY EXCEPT. 

It will raise Exception not return anything.